src/mlpack/methods/ann/layer/layer_types.hpp

```diff
@@ -256,6 +261,7 @@ using LayerTypes = boost::variant<
     MultiplyConstant<arma::mat, arma::mat>*,
     MultiplyMerge<arma::mat, arma::mat>*,
     NegativeLogLikelihood<arma::mat, arma::mat>*,
+    NoisyLinear<arma::mat, arma::mat>*,
```

Comment:
I moved `Glimpse<arma::mat, arma::mat>*` and `Highway<arma::mat, arma::mat>*,` into `using MoreTypes = boost::variant<` because one variant can only hold a specific number of layers.

Reply:
Thanks a lot! It works now. :)

Reply:
Great!